@xen-orchestra/web/src/components/vm/VmVifsTable.vue

```diff
@@ -0,0 +1,278 @@
+      </div>
+      <VtsDataTable :is-ready :has-error :no-data-message="vifs.length === 0 ? $t('no-vif-detected') : undefined">
+        <template #thead>
+          <tr>
+            <template v-for="column of visibleColumns" :key="column.id">
+              <th v-if="column.id === 'checkbox'" v-tooltip="$t('coming-soon')" class="checkbox">
+                <UiCheckbox :v-model="areAllSelected" accent="brand" disabled />
+              </th>
+              <th v-else-if="column.id === 'more'" class="more">
+                <UiButtonIcon v-tooltip="$t('coming-soon')" :icon="faEllipsis" accent="brand" disabled size="small" />
+              </th>
+              <th v-else>
+                <div v-tooltip class="text-ellipsis">
+                  <VtsIcon accent="brand" :icon="headerIcon[column.id]" />
+                  {{ column.label }}
+                </div>
+              </th>
+            </template>
+          </tr>
+        </template>
+        <template #tbody>
+          <tr
+            v-for="row of rows"
+            :key="row.id"
+            :class="{ selected: selectedVifId === row.id }"
+            @click="selectedVifId = row.id"
+          >
+            <td
+              v-for="column of row.visibleColumns"
+              :key="column.id"
+              class="typo-body-regular-small"
+              :class="{ checkbox: column.id === 'checkbox' }"
+            >
+              <div v-if="column.id === 'checkbox'" v-tooltip="$t('coming-soon')">
+                <UiCheckbox v-model="selected" accent="brand" :value="row.id" disabled />
+              </div>
+              <UiButtonIcon
+                v-else-if="column.id === 'more'"
+                v-tooltip="$t('coming-soon')"
+                :icon="faEllipsis"
+                accent="brand"
+                disabled
+                size="small"
+              />
+              <div v-else-if="column.id === 'status'" v-tooltip>
+                <VtsConnectionStatus :status="column.value" />
+              </div>
+              <div v-else-if="column.id === 'network'" class="network">
+                <!-- TODO Remove the span when the link works and the icon is fixed -->
+                <!--
+                  <UiComplexIcon size="medium" class="icon">
+                    <VtsIcon :icon="faNetworkWired" accent="current" />
+                    <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
+                  </UiComplexIcon>
+                  <a v-tooltip href="" class="text-ellipsis name">{{ column.value.name }}</a>
+                 -->
+                <span v-tooltip class="text-ellipsis name">{{ column.value }}</span>
+              </div>
+              <div v-else-if="column.id === 'ip'" class="ip-addresses">
+                <span class="value text-ellipsis">{{ column.value[0] }}</span>
+                <span v-if="column.value.length > 1" class="typo p3-regular more-ips">
```

Comment:
Old typography class:
```suggestion
                <span v-if="column.value.length > 1" class="typo-body-regular-small more-ips">
```

Reply:
oops